src/AppServices/KafkaPage/KafkaPage.tsx

```diff
@@ -35,50 +50,208 @@ export const KafkaPage: FunctionComponent = () => {
+                    components={{
+                      support: (
+                        <a
+                          href="https://access.redhat.com/support"
+                          target="_blank"
+                        />
+                      ),
+                      value: (
+                        <a
+                          href="https://access.redhat.com/documentation/en-us/red_hat_openshift_streams_for_apache_kafka/1/guide/aced8e5e-8229-4cb2-82f9-87a8caa24bb3"
+                          target="_blank"
+                        />
+                      ),
+                    }}
+                  />
+                }
+              </Text>
+            </TextContent>
+          </CardFooter>
+        </Card>
+      </MarketingPageSection>
+      <MarketingPageSection>
+        <Card>
+          <CardTitle>
+            <Title headingLevel="h3">{t("cloudProvidersTitle")}</Title>
+          </CardTitle>
+          <CardBody>
+            <Flex spaceItems={{ default: "spaceItemsXl" }}>
+              <Flex
+                alignSelf={{ default: "alignSelfCenter" }}
+                justifyContent={{ default: "justifyContentCenter" }}
+                flex={{ default: "flex_1" }}
+              >
+                <FlexItem>
+                  <Stack hasGutter>
+                    <Text component={TextVariants.p}>
+                      {t("aswDescription")}
+                    </Text>
+                    <Split hasGutter className="pf-u-p-xl">
+                      <SplitItem>
+                        <img
+                          src={LogoAWS}
+                          alt={""}
+                          style={{ height: "60px" }}
+                        />
+                      </SplitItem>
+                      <SplitItem>
+                        <Title headingLevel="h3">{"Amazon Web Services"}</Title>
+                      </SplitItem>
+                    </Split>
+                  </Stack>
+                </FlexItem>
+              </Flex>
+              <Divider isVertical />
+              <Flex
+                alignSelf={{ default: "alignSelfCenter" }}
+                justifyContent={{ default: "justifyContentCenter" }}
+                flex={{ default: "flex_1" }}
+              >
+                <FlexItem>
+                  <Stack hasGutter>
+                    <Text component={TextVariants.p}>
+                      {t("microsoftAzureDescription")}
+                    </Text>
+                    <Split hasGutter className="pf-u-p-xl">
+                      <SplitItem>
+                        <img
+                          src={LogoAzure}
+                          alt={""}
+                          style={{ height: "60px" }}
+                        />
+                      </SplitItem>
+                      <SplitItem>
+                        <Title headingLevel="h3">{"Microsoft Azure"}</Title>
+                      </SplitItem>
```

Comment:
Here is an image of the latest (and final) design for this section. Following this example, here are a couple of suggestions:

1. The "Coming soon" text can be moved to below the title. 
2. The "Coming soon" text should also have the className `pf-u-color-200` to make it gray.
3. The title shouldn't be perfectly top-aligned with the logo, but we also want to make sure the titles in both sections are aligned with each other. The best suggestion I have for this is to add the className `pf-u-pt-sm` to each title component to bump it down a bit. 


<img width="1135" alt="image" src="https://user-images.githubusercontent.com/21063328/165379862-3e3ee535-8685-4935-a3a0-11fc85b641cd.png">